angel_system/data/common/discretize_data.py

```diff
@@ -104,19 +114,27 @@ def discretize_data_to_windows(labels, gt, detections, time_window, uncertainty_
             time_windows,
         )
 
-        valid[ind1:ind2] = True
-        correct_label = detections["class_label"][i].strip().rstrip(".")
+        correct_label = detections["class_label"][i].lower().strip().strip(".").strip()
```

Comment:
Why are there so many `strip` calls? Are there really class labels like `  foo  .  `? Could this not be replaced by a single `.strip(' .')`? (not sure that your class label would have newlines in there, but you could add that too if needed...)

There are more instances of this in here.

Reply:
Yeah, there were a few instances of `foo  .  `. This is mainly trying to adjust for copy & paste spacing issues that we see a lot. I've at least set it up as a function so we can adjust how the strings are cleaned in one place. Would `.strip(" . ")` work instead?